.github/workflows/samples-php8.yaml

```diff
@@ -16,6 +16,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        php:
+          - "8.1"
+          - "8.2"
+          - "8.3"
```

Comment:
thanks for updating the workflow.

shall we test with 8.4 as well? or there're pending issues that need to be resolved before the output works with 8.4?

Reply:
@wing328 sorry i will fix soon

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/20168